src/studio-home/StudioHome.jsx

```diff
@@ -106,7 +111,7 @@ const StudioHome = ({ intl }) => {
     );
 
     return headerButtons;
-  }
+  }, [location]);
```

Comment:
When we add the actual create form, let's make it so this link uses React Router. Right now it causes a full page load, which is unnecessary.

Reply:
Fixed here: 4df024ecd1211a10638f40739a45f33b7b18bca7